dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.rest.client-3.0.feature

```diff
@@ -2,10 +2,10 @@
 symbolicName=io.openliberty.org.eclipse.microprofile.rest.client-3.0
 singleton=true
```

Comment:
didn't know where to put this comment, but io.openliberty.mpRestClient.internal.cdi-4.0.feature probably should be updated to beta instead of noship?

Reply:
Will have to coordinate with my PR though if you make that change.  Either need to rebase on integration after my change and update the test that will now fail, or I need to wait for your PR to merge and then update the test case before I merge.

Reply:
Sounds like it will be easier to do that as a separate change.

Reply:
I will add it to my PR now that you have merged this PR.